spec/features/categories_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe CategoriesController, type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+  let(:category) { FactoryBot.create(:category, user:) }
+  let(:transfers) { FactoryBot.create_list(:transfer, 6, author: user, category:) }
+
+  before(:each) do
+    sign_in user
+  end
+
+  describe 'Visits Index page' do
+    it 'Should see all the categories listed for the user' do
+      category
+      visit root_path
+      expect(page).to have_content(category.name)
+    end
```

Comment:
- [ ] one of the tests is failing please make sure to fix that in order to complete the requirements 
<img width="1120" alt="Screenshot 2023-02-15 at 15 47 32" src="https://user-images.githubusercontent.com/61790587/219061114-f4601b84-b96c-4df6-8fcb-f0691c796250.png">